Dockerfile.assisted-service

```diff
@@ -19,7 +19,7 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 RUN go mod download
 COPY . .
-RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service cmd/main.go
+RUN CGO_ENABLED=1 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service cmd/main.go
```

Comment:
If it remember correctly i used `CGO_ENABLED` to make the exec small.

Reply:
Explained here: https://github.com/openshift/assisted-service/pull/2233/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R162

Do you mean size of the binary? There is difference only 80kib in case of cgo enabled.

Reply:
OK i think that  now i remember it because we used `scratch` as a base image so we had to compile with this flag.